librz/core/disasm.c

```diff
@@ -2827,8 +2827,38 @@ static void ds_print_offset(RDisasmState *ds) {
 	}
 }
 
+static bool requires_op_size(RDisasmState *ds) {
+	RzPVector *metas = rz_meta_get_all_in(ds->core->analysis, ds->at, RZ_META_TYPE_ANY);
+	if (!metas) {
+		return false;
+	}
+
+	void **it;
+	bool res = true;
+	rz_pvector_foreach (metas, it) {
+		RzIntervalNode *node = *it;
+		RzAnalysisMetaItem *mi = node->data;
+		switch (mi->type) {
+		case RZ_META_TYPE_DATA:
+		case RZ_META_TYPE_STRING:
+		case RZ_META_TYPE_FORMAT:
+		case RZ_META_TYPE_MAGIC:
+		case RZ_META_TYPE_HIDE:
+		case RZ_META_TYPE_RUN:
```

Comment:
Btw, what is meta type `run`?

Reply:
```
			case RZ_META_TYPE_RUN:
				rz_core_cmd0(core, meta->str);
				break;
			default:
```
Some stuff...

Reply:
It's needed?

Reply:
... Who knows :(

Reply:
I have a feeling a corrupt project file is going to be lots of !!Fun!!.

Reply:
Who needs shellcode when you have this.

Reply:
I suggest to remove it.

Reply:
I'll do separate PR.